java/piranha/src/main/java/com/uber/piranha/config/Config.java

```diff
@@ -124,16 +126,33 @@ private Config(
   }
 
   /**
-   * Return all configuration method records matching a given method name.
+   * Return all configuration method records matching a given method name. A method record name
+   * field can be used to express a single method invocation or a chain of two method invocations. A
+   * chain of method invocations can be expressed by joining the name of the chained methods (from
+   * left to right) with a `.`. For instance, the name `isToggle` will match a invocation like -
+   * :[1].isToggle(:[2])), and the name `stale_flag.getValue` will match a invocation like -
```

Comment:
can we make this discussion more generic? This may be reachable to whoever follows the `hole` terminology.